src/data.js

```diff
@@ -0,0 +1,146 @@
+export default [
```

Comment:
Put this file into `fixtures` folder, also read about fixtures 

https://stackoverflow.com/questions/12071344/what-are-fixtures-in-programming